Dockerfile

```diff
@@ -1,27 +1,12 @@
-FROM circleci/node:10.14.2 as builder
-
-RUN sudo apt-get -y install --no-install-recommends libunwind8=1.1-4.1
-
-WORKDIR /usr/src/app
-
-COPY /src /usr/src/app/src
-COPY /package.json /usr/src/app/package.json
-COPY /tsconfig.json /usr/src/app/tsconfig.json
-COPY /yarn.lock /usr/src/app/yarn.lock
-
-RUN sudo chmod -R 777 /usr/src/app \
-  && yarn install \
-  && yarn build
-
 FROM node:10.14.2-alpine
 LABEL maintainer="https://pagopa.gov.it"
 
 WORKDIR /usr/src/app
 
 COPY /package.json /usr/src/app/package.json
-COPY --from=builder /usr/src/app/dist /usr/src/app/dist
-COPY --from=builder /usr/src/app/node_modules /usr/src/app/node_modules
+COPY /dist /usr/src/app/dist
```

Comment:
This means that we have to build locally before launch `docker compose` or `docker run` ?

Reply:
yes, in my opinion it's a better option